SwiftWisdomTests/Rx/Rx+ExtensionsTest.swift

```diff
@@ -6,6 +6,28 @@ import RxCocoa
 
 class OperatorTests: XCTestCase {
 
+    func testVariableBinding() {
+        let disposeBag = DisposeBag()
+        let label = UILabel()
+        let variable = Variable<String?>(nil)
+        XCTAssertEqual(nil, label.text)
+
+        label.rx.text <- variable >>> disposeBag
+        XCTAssertEqual(nil, label.text)
+
+        variable.value = "hello"
+        XCTAssertEqual("hello", label.text)
```

Comment:
could we also add an assert here and below to check that setting variable.value to nil sets label.text to nil as well?